data/json/item_groups.json

```diff
@@ -8603,6 +8603,20 @@
     ]
   },{
     "type" : "item_group",
+    "id" : "fuel_diesel",
+    "items":[
+      {"item": "diesel", "prob": 70, "charges-min": 950, "charges-max": 8000, "container-item": "jerrycan"},
+      {"item": "diesel", "prob": 30, "charges-min": 3850, "charges-max": 20000, "container-item": "jerrycan_big"},
```

Comment:
I've found this excessive comma after the PR creation, but it should be okay as-is, since cata doesn't blame me for that and works just fine during the test run.